src/interfaces/HXElement/index.js

```diff
@@ -262,7 +262,8 @@ export class HXElement extends HTMLElement {
      * @see https://goo.gl/MDp6j5
      */
     $upgradeProperty (prop) {
-        if (this.hasOwnProperty(prop)) {
+        let hasProp = Object.prototype.hasOwnProperty.call(this, prop);
```

Comment:
Correct errors as a result of [`no-prototype-builtins`](https://eslint.org/docs/rules/no-prototype-builtins) eslint rule.

Reply:
TODO: revert changes and update linting config